docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+---
+Order: 10
+Area: cpp
+TOCTitle: Get started with CMake Tools on Linux
+ContentId: 86543311-5452-4b1f-a44c-03cc3df04c3f
+PageTitle: Get started with CMake Tools on Linux 
+DateApproved: 06/11/2020
+MetaDescription: Get started with the CMake Tools Visual Studio Code extension on Linux 
+---
+# Get started with CMake Tools on Linux
+
+CMake is an open-source, cross-platform tool for building, testing, and packaging software. It uses platform and compiler independent configuration files to generate native makefiles that can be used with your compiler, on your platform.
```

Comment:
Nit: Makefiles is just one build system that CMake supports. Maybe 'generate native build tool files'?

Reply:
fixed in next commit.